src/dispatch/document/models.py

```diff
@@ -60,6 +69,12 @@ class Document(Base, ResourceMixin, TimeStampMixin):
     terms = relationship(
         "Term", secondary=assoc_document_terms, backref=backref("documents", cascade="all")
     )
+
+    evergreen = Column(Boolean)
+    evergreen_owner = Column(String)
```

Comment:
Should this be a relationship with individual contact? We could create the individual contact if one with the given email address does not exist.

Reply:
I thought about that, but I think it overcomplicates things. What if the owner is a team DL? Do we create a team contact or an individual contact? In the end, the notification doesn't care that we have individuals and contacts only that it should send a message to an email address.